app/services/albums.js

```diff
@@ -0,0 +1,13 @@
+const http = require('request-promise');
```

Comment:
I thinks the name is confusing, just call it request which is more descriptive of its functionality.